### PR TITLE
Fix Docroot Path for Minion

### PR DESCRIPTION
**Pull request by piotrgolasz**

### CODE REVIEW

**Comment by piotrgolasz** [comment]
@neo22s Why are tests not performing composer install?

**Comment by knbv** [comment]
#174

File: system/tests/kohana/MinionTaskTest.php [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ */
+class MinionTaskTest extends Unittest_TestCase
+{
+
+	/**
+	 * Tests that Minion Task Help works assuming all other tasks work aswell
+	 */
+	public function test_minion_runnable()
+	{
+		$minionResponse = Minion_Task::factory(['task' => 'help']);
```
Comment (knbv) [inline system/tests/kohana/MinionTaskTest.php]: Do I understand correctly that this test simply calls a task named "help"?

The fact is that the task will be executed regardless of whether the correct path is specified in the DOCROOT.

Comment (piotrgolasz) [reply system/tests/kohana/MinionTaskTest.php]: In unittest bootstrap i included autoloader.php (https://github.com/koseven/koseven/pull/194/files#diff-d97290f5a2964424212543942a6ede21R65)

Therefore if Minion task has executed correctly, that would mean that the docroot path is also correct.

Comment (knbv) [reply system/tests/kohana/MinionTaskTest.php]: Thanks you!

**Comment by knbv** [comment]
Sorry, I did not notice that you did not fix all the errors. The code does not match the Coding Style (String Concatenation).

https://docs.koseven.ga/guide/kohana/conventions

**Comment by knbv** [comment]
@piotrgolasz Sorry, I did not notice that you did not fix all the errors. The code does not match the Coding Style (String Concatenation).

https://docs.koseven.ga/guide/kohana/conventions

**Comment by piotrgolasz** [comment]
@knbv understood.

File: system/tests/kohana/MinionTaskTest.php (hunk 1 above)
Comment (errotan) [inline system/tests/kohana/MinionTaskTest.php]: $minionResponse does not conform with required coding style. It should be $minion_response.

File: modules/unittest/bootstrap.php [hunk 2]
```diff
@@ -61,6 +61,8 @@
  *
  * @link http://kohanaframework.org/guide/using.configuration
  */
+// Require composer libraries
+require DOCROOT.'/vendor/autoload.php';
```
Comment (errotan) [inline modules/unittest/bootstrap.php]: This change is not required for this patch.

File: .travis.yml [hunk 3]
```diff
@@ -21,6 +21,7 @@ before_install:
 # execute any number of scripts before the test run, custom env's are available as variables
 before_script:
   - sudo chmod -R 755 modules/ application/ system/
+  - composer install
```
Comment (errotan) [inline .travis.yml]: This change is not required for this patch.

**Pull request merged**